### PR TITLE
update dev local node docs

### PR DESCRIPTION
**Pull request by bthaile**
https://app.clubhouse.io/augur/story/7025/update-local-dev-node-instructions-scripts

update doc with flash commands and update running disco parrot via npx

### CODE REVIEW

File: docs/dev-local-node.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ We are going to use environment variables for convenience, ENDPOINT_HTTP and END
     export ETHEREUM_PRIVATE_KEY="fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a"
 
     # Use dp to deploy to the configuration specified in your local environment (above)
-    npm explore augur.js -- npm run deploy:environment
+    npx dp deploy
```
Comment (justinbarry) [inline docs/dev-local-node.md]: Should we replace `npm explore augur.js -- npm run deploy:environment` above with this?

File: docs/dev-local-node.md [hunk 2]
```diff
@@ -132,8 +135,7 @@ For more information on the script tool disco parrot (dp) [disco parrot](https:/
 
 Here are misc. script to do operations on the local node and to change time. More scripts will be created as they are needed, enivronment variables can be used or passed into the command like ETHEREUM_PRIVATE_KEY is in the examples below:
 
-    ** change time
-    ETHEREUM_PRIVATE_KEY=fae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5a node scripts/helpers/change-time.js <unix timestamp>
+For changing time on the contracts, use flash in augur.js:
 
     ** create open order
```
Comment (justinbarry) [inline docs/dev-local-node.md]: Should these three non-flash script items become a separate code block?

File: docs/dev-local-node.md (hunk 1 above)
Comment (bthaile) [reply docs/dev-local-node.md]: this is actually good, tested it.

File: docs/dev-local-node.md (hunk 2 above)
Comment (bthaile) [reply docs/dev-local-node.md]: I agree, I'll move down with more recent changes.

**Comment by bthaile** [comment]
the README.md can be updated to run new docker image for local node development. That would add value and talk about connecting to augur-node locally or in the cloud. I'll put it on my chore list in ch. 

https://app.clubhouse.io/augur/story/7112/update-augur-readme-md-to-clean-up

**Pull request merged**